### DISCUSSION TITLE
serenity ------> select row with radio button

### DISCUSSION THREAD

**Discussion by hosnabigdeli** [serenity-is/Serenity#5687] [Help]
hey
I have a column of radio buttons in the grid
How to get the value of that row from the table where the radio button was checked.

this my code (for show radio button in grid):
protected createToolbarExtensions() {
super.createToolbarExtensions();
this.radRowSelection = new Serenity.GridRadioSelectionMixin(this);
}

    protected getColumns() {
        var columns = super.getColumns();
        columns.splice(0, 0, Serenity.GridRadioSelectionMixin.createSelectColumn(() => this.radRowSelection));
    
        return columns;
    }
![44444444](https://user-images.githubusercontent.com/56981629/116843717-050d6300-abf6-11eb-84a7-7ebc7a6eff82.PNG)

**Comment by minhhungit** [comment]
@hosnabigdeli there is a wiki page here, it might help you 
https://github.com/serenity-is/Serenity/wiki/Grid:-Radio-Row-Selection-Mixin

![image](https://user-images.githubusercontent.com/2279508/116931649-73e8cb80-ac8b-11eb-9674-0f62b7076b5d.png)

**Reply by VictorTomaili** [reply]
Url changed because of file name restrictions on windows. 
https://github.com/serenity-is/Serenity/wiki/Grid-Radio-Row-Selection-Mixin
